src/main/java/com/minecolonies/structures/helpers/Structure.java

```diff
@@ -1,5 +1,10 @@
 package com.minecolonies.structures.helpers;
 
+import com.minecolonies.coremod.colony.ColonyManager;
+import com.minecolonies.coremod.colony.Structures;
+import com.minecolonies.coremod.configuration.Configurations;
+import com.minecolonies.coremod.lib.Constants;
+import com.minecolonies.coremod.MineColonies;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.minecolonies.coremod.MineColonies'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)